### DISCUSSION TITLE
Next Version ?

### DISCUSSION THREAD

**Discussion by walltztj** [rsjaffe/MIDI2LR#726] [General]
Anyone wondering if we will get a new version in the near future.

**Comment by D-Noc** [comment]
Wondering, yes.

But whether or not a new version will come soon is unknown (to me at least).
rsjaffe made the latest release back in July along with a post that he would be away from the project to at least September 22.
Now it is January 23 and he hasn't committed any changes or given any signs of activity, so it is hard to tell what will happen.

I don't know rsjaffe or what is happening with him, so it is hard to tell if he is caught up in other tasks or have personal stuff that requires his attention. After all the is working on this awesome software for free, so it may not be the most important thing in his life.

...and then there is the possibility that this software is now abandoned. I hope not.
I hope rjaffe returns or someone else picks up the ball

**Comment by antzpantz** [comment]
What would you be looking for in a new version anyway? Is there an issue?

**Reply by D-Noc** [reply]
Yes. Every time I choose Mask I get the error reported here:
https://github.com/rsjaffe/MIDI2LR/discussions/661#discussioncomment-3005897

**Comment by antzpantz** [comment]
I see, though the reply wasn't necessarily directed at you, mainly for
the OP of this thread.

**Reply by walltztj** [reply]
With the latest version of LR there are some changes that I would like to use and at the moment are not part of the Midi2LR program. Also there are some issues that users are having. If the OP has abandoned the project  it will be a sad day.

**Reply by D-Noc** [reply]
I agree. The LR API evolves with each release of LR. Even though I haven't yet had any new functrionality that I would like to map to a button/knob on my 2x XTouch Mini, I think that sooner or later something will change in the LR API and make MIDI2LR more and more obsolete.
That said I haven't looked at the source code. Depending on the way the application maps the API functions, it might be "just" a matter of adding/changing a configuration file.
